smaily_for_drupal/src/Form/SmailyForDrupalAdminForm.php

```diff
@@ -0,0 +1,273 @@
+<?php
+
+namespace Drupal\smaily_for_drupal\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Ajax\AjaxResponse;
+use Drupal\Core\Ajax\HtmlCommand;
+use Drupal\Core\Ajax\ReplaceCommand;
+
+/**
+ * Smaily module admin configuraiton form.
+ */
+class SmailyForDrupalAdminForm extends ConfigFormBase {
```

Comment:
I think this can just be `AdminForm`. The `namespace` value should make sure that this does not conflict with any other module.